Fix snippet example in stream-rate doc

atamborrino opened the pull request:
Fix #section-buffer by replacing .withAttributes by .addAttributes to avoid overwriting the .async option.

atamborrino committed:
Update StreamBuffersRateSpec.scala

Fix #section-buffer by replacing .withAttributes by .addAttributes to avoid overwriting the .async option.

typesafehub-validator commented:
Hi @atamborrino,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Typesafe Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

atamborrino committed:
Update StreamBuffersRateDocTest.java

Fix #section-buffer by replacing .withAttributes by .addAttributes to avoid overwriting the .async option.

ktoso commented:
Nice catch, thanks!

ktoso commented:
OK TO TEST

johanandren commented:
Unrelated test failure

johanandren commented:
PLS BUILD

johanandren commented:
LGTM

johanandren commented:
Issue: #21006
